sunpycube/cube/NDCube.py

```diff
@@ -71,12 +69,8 @@ def plot(self, *args, **kwargs):
     def __getitem__(self, item):
         if item is None or (isinstance(item, tuple) and None in item):
             raise IndexError("None indices not supported")
-        return self._getitem_NDCube(item)
-
-    def _getitem_NDCube(self, item):
         data = self.data[item]
-        # wcs = self.wcs.wcs_slicer[item]
-        wcs = None
+        wcs = self.wcs.wcs_slicer[item]
```

Comment:
my point was that it should not be an attribute of the WCS class at all. Because it restricts the type of the WCS object. I.e. it should be possible to have a `astropy.wcs.WCS` object as long as you don't need the axis augmentation stuff in the cube subclass.